composer.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "netgen/ibexa-scheduled-visibility",
+  "description": "Netgen's scheduled visibility mechanism for Ibexa CMS",
+  "type": "ibexa-bundle",
+  "license": "GPL-2.0-only",
+  "authors": [
+    {
+      "name": "Netgen",
+      "homepage": "https://netgen.io"
+    }
+  ],
+  "require": {
+    "php": ">=8.1",
+    "ibexa/core": "^4.6"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^8.5",
```

Comment:
Is there a reason why we don't use PHPUnit 10 or 11 here?

Reply:
no, bumped to version 10 in de46a455543ad66162d7145f07b5e278e9982fc6